Makefile

```diff
@@ -323,10 +324,15 @@ telemetrygen:
 .PHONY: update-otel
 update-otel:$(MULTIMOD)
 	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m stable --commit-hash $(OTEL_STABLE_VERSION)
-	git add . && git commit -s -m "[chore] multimod update stable modules"
+	git add . && git commit -s -m "[chore] multimod update stable modules" ; \
 	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m beta --commit-hash $(OTEL_VERSION)
-	git add . && git commit -s -m "[chore] multimod update beta modules"
+	git add . && git commit -s -m "[chore] multimod update beta modules" ; \
+	sed -i -E -e "s/v[0-9]+\.[0-9]+\.[0-9].*$\/v$(TRIMMED_VERSION)/" -e "5,6s/[0-9]+\.[0-9]+\.[0-9]/$(TRIMMED_VERSION)/" ./cmd/otelcontribcol/builder-config.yaml 
```

Comment:
It's possible I'm reading this wrong, but this will update the builder config with a trimmed version produced from this repo? something like `v0.97.0` correct? 

What we really want here is for the builder-config version for core dependencies (any of the deps listed in [versions.yaml](https://github.com/open-telemetry/opentelemetry-collector/blob/main/versions.yaml)) to match their versions in the go.mod file. See an example here: 

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d936776c102f185cdf6aa7604320fcdb79f62588/go.mod#L169

Reply:
yes that is correct


> What we really want here is for the builder-config version for core dependencies (any of the deps listed in [versions.yaml](https://github.com/open-telemetry/opentelemetry-collector/blob/main/versions.yaml)) to match their versions in the go.mod file. See an example here:

i'm going to have a look into this

Reply:
Thanks @led0nk! It's a really annoying problem 😆